content/news/2022-07-29-bevy-0.8/index.md

```diff
@@ -0,0 +1,1131 @@
++++
+title = "Bevy 0.8"
+date = 2022-07-29
+[extra]
+author = "Carter Anderson"
+twitter = "cart_cart"
+github = "cart"
+youtube = "cartdev"
+image = "bevy_terrain.png"
+show_image = true
+image_subtitle = "Bevy-shaped mountains in a Bevy-based Witcher 3 terrain texturing tool built by rmemr"
+image_subtitle_link = "https://codeberg.org/rmemr/w3.terrain-texturing"
++++
+
+Thanks to **X** contributors, **X** pull requests, and our [**generous sponsors**](https://github.com/sponsors/cart), I'm happy to announce the **Bevy 0.8** release on [crates.io](https://crates.io/crates/bevy)!
+
+For those who don't know, Bevy is a refreshingly simple data-driven game engine built in Rust. You can check out [Quick Start Guide](/learn/book/getting-started/) to get started. Bevy is also free and open source forever! You can grab the full [source code](https://github.com/bevyengine/bevy) on GitHub. Check out [Bevy Assets](https://bevyengine.org/assets) for a collection of community-developed plugins, games, and learning resources.
+
+To update an existing Bevy App or Plugin to **Bevy 0.8**, check out our [0.7 to 0.8 Migration Guide](/learn/book/migration-guides/0.7-0.8/).
+
+As always, there are a _ton_ of new features, bug fixes, and quality of life tweaks in this release, but here are some of the highlights: 
+
+* **New Material system**: Custom shaders are now _much_ easier to define, thanks to the new Material trait and AsBindGroup derive
+* **Camera driven rendering**: Each Camera now configures what it renders and how it renders it. Easily layer camera renders on top of each other, do split screen, or render to a texture in just a few lines of code.
+* **Built-in Shader Modularization**: Many built in shader types and functions are now importable. Notably, custom shaders can now import the PBR shader logic
+* **Spot Lights**: A new light type that emits light in a cone shape from a fixed point
+* **Visibility Inheritance**: Hiding an entity now also hides all of its descendants in the hierarchy
+* **Scene Bundle**: Easily spawn scenes using a normal Bevy bundle and extend them with new components and children
```

Comment:
This should be bumped down out of the rendering section.

Reply:
Agree.